baseapp/files/services/upload_service.py

```
@@ -0,0 +1,163 @@
+    def abort_multipart_upload(self, file_id: int):
+        """
+        Abort a multipart upload and cleanup.
+        """
+        file_obj = File.objects.select_for_update().get(id=file_id)
+
+        if file_obj.upload_id:
+            try:
+                self.handler.abort_upload(file_obj, file_obj.upload_id)
+            except Exception:
+                # Log but continue to mark as aborted
+                pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)